Longest_Increasing_Subarray/Longest_Increasing_Subarray.py

```diff
@@ -0,0 +1,49 @@
+'''
+    LONGEST INCREASING SUBARRAY
+
+    The task is to find a subarray from a given array having the maximum length
+    such that the elements of the subarray are in ascending order.
+'''
+
+'''
+    Approach : If the current element is greater than the last element, we 
+               update the length variable, so as to keep a counter of the 
+               maximum numbers in the current longest increasing subarray.
+               The mx variable keeps the track of the maximum length across
+               all increasing subarrays found.
+               The max_index denotes the starting index of the longest 
+               increasing subarray found.
+               We reset the length to 1 whenever the current element is found
+               to be lesser than the previous element.
+'''
+
+mx = 0
+ln = 1
+max_index = 0
+
+def solve(a, n) :
```

Comment:
Please have a look at space before `:` in python.

Reply:
Should I not give it ?
It works fine.